bcda/bcdacli/cli.go

```diff
@@ -170,6 +172,57 @@ func setUpApp() *cli.App {
 				return nil
 			},
 		},
+		{
+			Name: "save-public-key",
+			Category: "Authentication tools",
+			Usage: "Upload an ACO's public key to the database",
+			Flags: []cli.Flag{
+				cli.StringFlag{
+					Name:        "cms-id",
+					Usage:       "CMS ID of ACO",
+					Destination: &acoCMSID,
+				},
+				cli.StringFlag{
+					Name: "key-file",
+					Usage: "Location of public key in PEM format",
+					Destination: &filePath,
+				},
+			},
+			Action: func(c *cli.Context) error {
+				if acoCMSID == "" {
+					fmt.Fprintf(app.Writer, "cms-id is required\n")
+					return errors.New("cms-id is required")
+				}
+
+				if filePath == "" {
+					fmt.Fprintf(app.Writer, "key-file is required\n")
+					return errors.New("key-file is required")
+				}
+
+				aco, err := auth.GetACOByCMSID(acoCMSID)
+				if err != nil {
+					fmt.Fprintf(app.Writer, "Unable to find ACO %s: %s\n", acoCMSID, err.Error())
+					return err
+				}
+
+				/* #nosec -- Potential file inclusion via variable */
```

Comment:
This line is unnecessary now that you have the filepath.Clean command in the line below.  I ran it locally to confirm.

Reply:
OK.